layouts/gate.njk

```diff
@@ -1,23 +1,28 @@
 {% extends "base.njk" %}
 
-
 {% block content %}
-<h2>{{ title }}</h2>
+<div class="gate">
+  <h2 class="gate-title">{{ title }}</h2>
 
-<p><strong>Symbol:</strong> ${{ symbol }}$</p>
+  <div class="symbol-row">
+    {#<p><strong>Symbol:</strong> ${{ symbol }}$</p> #}
+    <p>Symbol</p>
+    <div class="symbol-box">${{ symbol }}$</div>
+  </div>
```

Comment:
This is awesome, and I love the periodic table inspiration.

It makes me think the homepage should be a grid of boxes with these instead of just a list. We could take it a step further, and add some more information to each box, such as the arity, and group it lives inside. We could number them as well, but it would be arbitrary.